docs/post/memcards.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Memory Cards"
+date: 2024-03-20
+summary: "This guide shows you how to import save data to your virtual memory card"
+draft: false
+toc: true
+sidebar_position: 4
+---
+
+This guide shows you information about Memory Cards in general.
+
+## General
+
+When you first install PCSX2, it comes with two virtual memory cards that you can use right away. You can also create a new one and even delete any of the existing ones.
+
+<Image src={require("./img/memcard.webp").default} />
+
+### Memory Cards Types
+
+There are two types of Memory Cards in PCSX2; File Memory Card and Folder Memory Cards.
+
+<Image src={require("./img/memcard_create.webp").default} />
+
+#### File Memory Card
+
+File Memory Cards are what you would usually expect from a standard Memory Cards, they are a single `.ps2` file and they stores all the save data inside that file for that particular card. they are also available in multiple sizes, 8MB, 16MB, 32MB and 64MB. 8MB is the most compatible option. If you need a bigger card capacity, refer to Folder Memory Card.
```

Comment:
"File Memory Cards are what you would usually expect from a standard Memory Cards, they are a single" reads a bit awkwardly, maybe "File Memory Cards are the standard, most compatible memory card type. They are a single"

Missed capitalization on "they are".

"multiple sizes," should be "multiple sizes:".